### ISSUE TITLE
Core is broken

### ISSUE THREAD

**Issue by sergiobenrocha2** [libretro/libretro-meta#59]
_From @fr500 on November 6, 2015 14:2_

Seems to me since
https://github.com/libretro/gambatte-libretro/commit/75dca159608602252cbd693cbac763077898f65b

Tried only on win x64

_Copied from original issue: libretro/gambatte-libretro#45_

**Comment by sergiobenrocha2** [comment]
_From @hizzlekizzle on November 6, 2015 14:23_

"Broken" as in "crashing immediately"? If so, I'm not having any issues on win64 with the latest from the buildbot.

**Comment by sergiobenrocha2** [comment]
_From @fr500 on November 6, 2015 14:24_

yeah it's crashing at startup

**Comment by sergiobenrocha2** [comment]
_From @fr500 on November 6, 2015 14:26_

oh... it seems to be cause by retroachievements... never mind then 
@leiradel

**Comment by sergiobenrocha2** [comment]
_From @leiradel on November 6, 2015 15:22_

Which game were you trying?

**Comment by sergiobenrocha2** [comment]
_From @leiradel on November 6, 2015 15:37_

I tried with **Super Mario Land 2** and it's crashing in `retro_get_memory_size`, which is indeed called by `cheevos_load` when it checks if the core supports cheevos or not.

It seems this core is not fully initialized when `retro_load_game` is called which is understandable since many things are only known by the core when the it receives the content it has to play.

I've changed `content.c` so that `cheevos_load` is called _after_ `retro_load_game` and it seems to solve the problem.
